README.md

```diff
@@ -28,7 +28,33 @@ Install Requirements - Start HTTP Server. After running the following playbook y
 $ ansible-playbook setup.yml --ask-sudo-pass
 ```
 
-**Replace the Variables in `vars/atomic.yml` with your httpserver IP Address and OSTree name.**
+**Replace the Variables in `vars/atomic.yml` with your httpserver IP Address, OSTree name, Basehost.**
+
+**If you wish to use CentOS Atomic modify the variables accordingly.**
+
+```
+---
+# Variables for Atomic host
+atomicname: my-atomic
+basehost: fedora-atomic/25/x86_64/docker-host
+httpserver: 192.168.122.1
+```
+
+**Add the additional Packages you would like to have in the OSTree in `vars/buildrepo.yml`.**
+
+**If you wish to use CentOS Atomic modify the variables accordingly.**
+
+```
+---
+repo: https://pagure.io/fedora-atomic.git
+branch: f25
+repodir: fedora-atomic
+abs_path: /workspace                                  # The absolute path to the git repo.
+coremanifest: customized-atomic-docker-host.json      # The manifest that goes into the custom host(ostree) content that we are going to build.
```

Comment:
but in this context it is a derived version. coremanifest can therefore be misunderstood for it's purpose.